### PR TITLE
Update StashDB details in README.md

### PR DESCRIPTION
**Pull request by AdultSun**
- Directs users to new guide in the StashDB docs instead of Discord
- No longer necessary to join Discord/Matrix for new users of StashDB now that invite codes are multi-use
- Updates formatting of the same "Quickstart Guide" section a little

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,9 +48,9 @@ Stash is a web-based application. Once the application is running, the interface
...
 
-Stash can pull metadata (performers, tags, descriptions, studios, and more) directly from many sites through the use of [scrapers](https://github.com/stashapp/stash/tree/develop/ui/v2.5/src/docs/en/Scraping.md), which integrate directly into Stash.
-
-Many community-maintained scrapers are available for download from [CommunityScrapers repository](https://github.com/stashapp/CommunityScrapers). The community also maintains StashDB, a crowd-sourced repository of scene, studio, and performer information, that can automatically identify much of a typical media collection. Inquire in the Discord for details. Identifying an entire collection will typically require a mix of multiple sources. 
+Stash can pull metadata (performers, tags, descriptions, studios, and more) directly from many sites through the use of [scrapers](https://github.com/stashapp/stash/tree/develop/ui/v2.5/src/docs/en/Scraping.md), which integrate directly into Stash. Identifying an entire collection will typically require a mix of multiple sources:
+- Many community-maintained scrapers are available for download from [CommunityScrapers repository](https://github.com/stashapp/CommunityScrapers).
```
Comment (DogmaDragon) [inline README.md:52]: I would suggest including mention that you can install them directly from Stash. Something to the effect of:
- Many community-maintained scrapers in [CommunityScrapers repository](https://github.com/stashapp/CommunityScrapers) can be installed directly from Stash (Settings > Metadata Providers). For more details check the repository [README.md](https://github.com/stashapp/CommunityScrapers/blob/master/README.md).

File: README.md [hunk 2]
```diff
@@ -48,9 +48,9 @@ Stash is a web-based application. Once the application is running, the interface
...
+- The community also maintains [StashDB](https://stashdb.org/), a crowd-sourced repository of scene, studio, and performer information, that can automatically identify much of a typical media collection. Instructions, invite codes, and more can be found in this guide to [Accessing StashDB](https://guidelines.stashdb.org/docs/faq_getting-started/stashdb/accessing-stashdb/).
```
Comment (DogmaDragon) [inline README.md:53]: Using scrapers is more technical than setting up StashDB, maybe worth switching the order where StashDB is first and scrapers second?

Comment (DogmaDragon) [inline README.md:53]: I would add that StashDB is focused on mainstream scenes and studios.

**Comment by AdultSun** [comment]
Thanks, I've incorporated your suggestions and added links to the guide for non-StashDB stash-boxes and the "Guide to Scraping" at the bottom for further explanation on how to actually start scraping stuff.

That last guide still needs to be updated and expanded too. If you don't have any plans to tackle it in the next week or so, I can start working on it soon as my next documentation project.

**Pull request merged**